src/input.cpp

```diff
@@ -1355,6 +1357,35 @@ cata::optional<tripoint> input_context::get_coordinates( const catacurses::windo
 }
 #endif
 
+std::tuple<point, bool> input_context::get_coordinates_text( const catacurses::window
```

Comment:
is there a reason this is a tuple and not a pair?

Reply:
No reason, but does it matter?